bigquery/google/cloud/bigquery/job.py

```diff
@@ -1949,6 +1953,29 @@ def result(self, timeout=None, retry=DEFAULT_RETRY):
         return self._client.list_rows(dest_table, selected_fields=schema,
                                       retry=retry)
 
+    def to_dataframe(self):
+        """Create a pandas DataFrame from the query results.
+
+        :rtype: ``pandas.DataFrame``
```

Comment:
This should be:

```
:class:`pandas.DataFrame`
# OR
:class:`~pandas.DataFrame`
```

It will even render as a link to the pandas docs because we have `pandas` in [our intersphinx mapping][1].

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/69a9840955caefbd431ee459ae91632b9cf79a16/docs/conf.py#L302